trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/base/library/AtomicLibraryModel.java

```diff
@@ -73,10 +74,8 @@
  * @author Dominik Klumpp (klumpp@informatik.uni-freiburg.de)
  */
 public class AtomicLibraryModel implements ILibraryModel {
-	/**
-	 * See MEMORY_ORDER_SEQ_CST in stdatomic.h
-	 */
-	private static final int MEMORY_ORDER_SEQ_CST = 5;
+	private static final String[] MEMORY_ORDER = { "memory_order_relaxed", "memory_order_consume",
+			"memory_order_acquire", "memory_order_release", "memory_order_acq_rel", "memory_order_seq_cst" };
```

Comment:
Representing memory orders as a string array can be dangerous, since the `memory_order` from `stdatomic.h` is represented as an `enum`. Wouldn't it be better to use an `enum` with an attached string value in Java?

Reply:
To define the `CEnum` `memory_order`, I need exactly such a string array, therefore this was the simplest solution.
If I would use an enum instead, this would be more complex and I cannot really see the advantage.